wgengine/filter/filter.go

```diff
@@ -352,7 +352,7 @@ func (f *Filter) runIn4(q *packet.Parsed) (r Response, why string) {
 		if f.matches4.match(q) {
 			return Accept, "tcp ok"
 		}
-	case packet.UDP:
+	case packet.UDP, packet.SCTP:
```

Comment:
Use a different LRU for each protocol, otherwise an outbound SCTP packet would allow UDP return.

Reply:
(or add the proto to the flowtrack tuple - 5-tuple of proto,srcip,dstip,srcport,dstport is a common network thing, we should have one)

Reply:
I went with the 5-tuple before I saw your follow-up comment, so I feel like that's a good answer. :)

It ended up making things nicer in other places too, like TSMP rejected header's Flow method no longer loses that info when returning the flowtrack.Tuple.